dask/dataframe/core.py

```diff
@@ -4672,15 +4673,82 @@ def set_index(
         >>> # ^ Now copy-paste this and edit the line above to:
         >>> # ddf2 = ddf.set_index("name", divisions=["Alice", "Laura", "Ursula", "Zelda"])
         """
+
+        # Deal with bool params
```

Comment:
I don't think this comment adds any understanding. Can you make it longer to explain why this is being done here, or remove it?